Infrastructure: tidy up naming of SLOT methods and their usage - Part 4

SlySven opened the pull request:
When Qt's slot/signal system is used to invoke a method there is some overhead - so it makes sense to ensure developers can spot all such methods (functions). We have tried to do this with a `slot_` prefix to the methods we create but it has not been applied uniformly. This PR (and one or more to follow) is intended to help with this by more rigorously doing so - the names changed herein should all follow a `slot_`*camelCaseMethodName* style. In addition some methods were detected that are not currently used or which are not actually used via the signal/slot system, these have been commented out or have had the prefix removed and the declaration in the relevant header file moved as appropriate.

For reference the changes made are:
* `dlgAliasMainArea::slot_editing_name_finished()` ==> `dlgAliasMainArea::slot_editingNameFinished()`
* `dlgScriptsMainArea::slot_editing_name_finished()` ==> `dlgScriptsMainArea::slot_editingNameFinished()`
* `dlgScriptsMainArea::slot_editing_event_name_finished()` ==> `dlgScriptsMainArea::slot_editingEventNameFinished()`, *added in a later commit.*
* `dlgTimersMainArea::slot_editing_name_finished()` ==> `dlgTimersMainArea::slot_editingNameFinished()`
* `dlgTriggersMainArea::slot_editing_name_finished()` ==> `dlgTriggersMainArea::slot_editingNameFinished()`

It ~~seems~~*was* odd that there ~~are~~*were* not corresponding `slot_`s to trim spaces off either end for the "Actions" (Buttons/Menus/Toolbars) and "Keybindings" Mudlet item types, *so they have been added as well.*

Signed-off-by: Stephen Lyons <slysven@virginmedia.com>

SlySven committed:
Infrastructure: tidy up naming of SLOT methods and their usage - Part 4

When Qt's slot/signal system is used to invoke a method there is some
overhead - so it makes sense to ensure developers can spot all such
methods (functions). We have tried to do this with a `slot_` prefix to the
methods we create but it has not been applied uniformly. This PR (and
one or more to follow) is intended to help with this by more rigorously
doing so - the names changed herein should all follow a
`slot_`*camelCaseMethodName* style. In addition some methods were detected
that are not currently used or which are not actually used via the
signal/slot system, these have been commented out or have had the prefix
removed and the declaration in the relevant header file moved as
appropriate.

For reference the changes made are:
* `dlgAliasMainArea::slot_editing_name_finished()`
                         ==> `dlgAliasMainArea::slot_editingNameFinished()`
* `dlgScriptsMainArea::slot_editing_name_finished()`
                       ==> `dlgScriptsMainArea::slot_editingNameFinished()`
* `dlgTimersMainArea::slot_editing_name_finished()`
                        ==> `dlgTimersMainArea::slot_editingNameFinished()`
* `dlgTriggersMainArea::slot_editing_name_finished()`
                      ==> `dlgTriggersMainArea::slot_editingNameFinished()`

It seems odd that there are not corresponding `slot_`s to trim spaces off
either end for the "Actions" (Buttons/Menus/Toolbars) and "Keybindings"
Mudlet item types.

Signed-off-by: Stephen Lyons <slysven@virginmedia.com>

vadi2 commented:
> It seems odd that there are not corresponding slot_s to trim spaces off either end for the "Actions" (Buttons/Menus/Toolbars) and "Keybindings" Mudlet item types...

That looks like an oversight

SlySven committed:
Revise: rename extra slot used in Script items

Which I missed from previous commit:
* `dlgScriptsMainArea::slot_editing_event_name_finished()`
                  ==> `dlgScriptsMainArea::slot_editingEventNameFinished()`

Signed-off-by: Stephen Lyons <slysven@virginmedia.com>

SlySven committed:
Fix: add name trimming to Keys and Buttons/Menus/Toolbars like other items

This is a response to:
https://github.com/Mudlet/Mudlet/pull/6259#issuecomment-1229194838

Whilst doing this I also spotted an issue which has been marked with a
TODO, and so it does not cause a "Danger" to this PR that has been raised
as #6261 !

Signed-off-by: Stephen Lyons <slysven@virginmedia.com>

SlySven commented:
> > It seems odd that there are not corresponding slot_s to trim spaces off either end for the "Actions" (Buttons/Menus/Toolbars) and "Keybindings" Mudlet item types...
> >
> That looks like an oversight

Fixed in https://github.com/Mudlet/Mudlet/pull/6259/commits/70944d7bbb38da07c903175b2afaae33b68bcb4a .
